extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/HealthBuildTimeConfig.java

```diff
@@ -5,6 +5,12 @@
 
 @ConfigRoot(name = "health")
 public class HealthBuildTimeConfig {
+    /**
+     * Activate or disable this extension. Disabling this extension means that no health related information is exposed.
+     */
+    @ConfigItem(name = "active", defaultValue = "true")
+    public boolean active;
```

Comment:
If you disable the extension entirely, please use `enabled`, not `active`.

Also don't define a `name`, the convention will make it right.

See this issue for the overall context: https://github.com/quarkusio/quarkus/issues/42244